assets/scripts/google-analytics-opt-out.js

```diff
@@ -14,7 +14,7 @@ function gaOptout() {
   var disableStr;
   for (var i = 0; i < window.gaIdList.length; i++) {
     disableStr = 'ga-disable-' + window.gaIdList[i];
-    document.cookie = disableStr + '=true; expires=Thu, 31 Dec 2099 23:59:59 UTC; path=/';
+    document.cookie = disableStr + '=true; domain=bnn.de; expires=Thu, 31 Dec 2099 23:59:59 UTC; path=/';
```

Comment:
The custom domain cannot be hardcoded. The backend would need to pass the value to use for the cookie domain as a JS setting.